manage-participants/completion_callback.yml

```diff
@@ -6,6 +6,14 @@
 - name: Completion Callback
   gather_facts: false
   hosts: localhost
+  vars:
+    # default __meta__ to prevent errors on older ansible versions
+    __meta__:
+      callback: {}
+    agnosticd_callback_url: >-
+      {{ __meta__.callback.url | default('') if __meta__ is defined and __meta__.callback is defined else '' }}
```

Comment:
the `default('')` should be all you need here - i.e.: no need for the if/else logic

Reply:
same below

Reply:
Done